arrow-site/docs/_data/sidebar-quickstart.yml

```
@@ -0,0 +1,25 @@
+options:
+  - title: Introduction
+    url: /quickstart/#rrow-is-a-library-for-typed-functional-programming-in-kotlin
+
+  - title: Setup
+    url: /quickstart/#setup
+
+  - title: Media
+    url: https://media.arrow-kt.io/tags/core/
```

```suggestion
    url: https://media.arrow-kt.io
```

All the tags for the link in the QuickStart :raised_hands: